src/components/ProductDetail.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import StarRating from "./StarRating";
+
+function ProductDetail(props){
+    let items = props.items;
+    let itemDivs = items.map(function (item, index) {
+      return (
+
+          <div>
```

Comment:
I actually had the key and deleted it because I had an issue with items not showing after I put in the map function.  So started debugging some things and forgot to put it back in!

I had to change the way state.js gave us the items similar to how they were in the other exercise.  How do you access them if it is already a named array like it is in state.js and not a default array like it was in our previous contact list example.

I imagine we just call it by its name?  like State.nameofarray?